spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/RuntimeAppsController.java

```diff
@@ -90,6 +90,16 @@ public AppStatusResource display(@PathVariable String appId) {
 		return statusAssembler.toResource(status);
 	}
 
+	@RequestMapping("{streamName}/logs")
```

Comment:
You can't have this as `logs` is expected to be an `appName` for mapping just below.

Reply:
And we already have `/{appId}` at root, so you can't just switch to `streamName` as it gets confusing. `RuntimeAppController` is really only meant for apps, not streams.

Reply:
Yeah, missed that. Now the PR is updated with the changes which move the new REST endpoints into separate controllers.

Reply:
These should then go away now that you moved it to a new controller?